migrations/handle_subscriptions.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ *
+ * phpBB Browser Push Notifications. An extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2023, phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\webpushnotifications\migrations;
+
+use phpbb\db\migration\container_aware_migration;
+
+class handle_subscriptions extends container_aware_migration
+{
+	public function revert_data(): array
```

Comment:
We should probably have an effectively installed method too, so we make sure this only runs if our extension is installed (so in other words this won't run during an installation, only during a delete/purge). 

Maybe:
```php
public function effectively_installed()
{
	return !$this->db_tools->sql_table_exists($this->table_prefix . 'wpn_notification_push');
}
```